pontoon/sync/files.py

```diff
@@ -0,0 +1,15 @@
+"""
+Various functions related to the paths and the io related stuff.
+"""
+
+import os
+
+
+def is_hidden(path):
+    """
+    Return true if path contains hidden directory.
+    """
+    for p in path.split(os.sep):
+        if p.startswith('.'):
+            return True
+    return False
```

Comment:
Actually there was an circular import before the 41e1a2b. However, it doesn't seem to be a case currently. I'll fix this.